MiniApp/Classes/Display/MiniAppWebView.swift

```diff
@@ -3,13 +3,14 @@ import WebKit
 internal class MiniAppWebView: WKWebView {
 
     convenience init?(filePath: URL) {
+        let fileURL = filePath.appendingPathComponent(Constants.FileNames.baseFile)
```

Comment:
I'm wondering if maybe `index.html` should just be inline here instead of in a constants file? I can't think of any situation right now where anything other than `MiniAppWebView` needs to know about `index.html`.

Reply:
🤔 Could be. I was thinking like a standard practise to have constants for file names